scripts/prepare_client.sh

```diff
@@ -76,6 +76,7 @@ vrf:    \"${VRF}\"
 kt-key: \"${KTKEY}\"
 kt-sig: \"${SIGKEY}\"
 domain: \"${DOMAIN}\"
+mapid: 0 
```

Comment:
Can you also make the same change in the checked-in default `.keytransparency.yaml` file?